style.css

```diff
@@ -88,21 +72,314 @@ h3 {
+  border: 1px solid black;
+}
+
+.project {
+  padding: 10px;
+  background-image: url(Images/Background.png);
+  background-position: center;
+  background-size: no-repeat;
+  margin: 25px;
+}
+
+.contenthead {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 44px;
+  color: #fdf6f6;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-top: 25%;
+}
+
+.projectlinks {
+  display: flex;
+  width: 85%;
+  justify-content: space-around;
+  padding: 5px;
+}
+
+.projectlinks li {
+  display: flex;
+  justify-content: space-around;
+  padding: 5px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 200;
+  font-size: 12px;
+  line-height: 20px;
+  color: rgb(245, 240, 240);
+  border: 0.5px solid rgb(250, 244, 244);
+  list-style-type: none;
+  width: 30%;
+  background-color: grey;
+}
+
+.btn {
+  width: 100%;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: hsl(24, 29%, 97%);
+  background-color: #f65803;
+}
+
+.aboutsection {
+  background-color: #00040a;
+  padding: 30px;
+  background-image: url(Images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: right 270px;
+  padding-bottom: 10px;
+}
+
+.about-part {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2px;
+  gap: 34px;
+  text-align: center;
+}
+
+.aboutmehead {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'Crete Round';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 55px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+}
+
+.container2 {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: repeat(6, 3px);
+  row-gap: 15px;
+}
+
+.buttn {
+  border: none;
+  color: #edeae9;
+  padding: 20px 36px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 20px;
+  background-color: #f65803;
+}
+
+.line {
+  color: rgb(251, 242, 242);
+  border-bottom: none;
+  margin-bottom: 100%;
+}
+
+.tools {
+  position: relative;
+  top: -200px;
+}
+
+.toolhead {
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: 'InaiMathi';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 29px;
+  color: hsl(0, 0%, 100%);
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  background: #00040a;
+  border-bottom: 1px solid black;
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
Kindly make sure that your design sticks as much as possible to the Figma template provided. The technologies section is ordered in a different way. :recycle: :recycle: 
![image](https://user-images.githubusercontent.com/98363075/192100848-2b1c2771-db4b-4f28-935f-80e0cc355dd9.png)